src/controllers/heroPostController.js

```diff
@@ -0,0 +1,20 @@
+const { addHero } = require('../services/herolist');
+
+const heroPostController = (req, res) => {
+  const { userId, heroName } = { userId: Number(req.headers.userid), heroName: req.headers.name };
+  if (userId && heroName) {
+    addHero(heroName, userId)
+      .then((response) => {
+        res.status(200).json(response);
+      })
+      .catch((error) => {
+        res.status(400).json(error);
```

Comment:
This is not necessarily a 400 error, it's more likely a 500 error.

Reply:
Fixed thank you!